neuralmonkey/tf_manager.py

```diff
@@ -236,16 +229,17 @@ def execute(self,
             A list of `ExecutionResult` tuples, one for each executable
             (runner).
         """
+        feedables = set.union(*[runner.feedables for runner in runners])
+        default_feed_dict = _feed_dicts(batch, feedables, train=train)
```

Comment:
Takže jsi zahodil ty additional feed dicty?

Reply:
tadyty dva řádky jsou jen refactor, díky kterýmu `_run_executables` nemusí žrát `batch`, `feedables` a `train`.

Mění se pořadí tvorby feed dictů: Bylo to tak, že se napřed (v `_run_executables`) vygenerovaly additional feed dicty z runnerů a pak se updatovaly feed dictama z feedables. Teď je to tak, že se napřed (už v týhle funkci) vygenerujou feed dicty z feedables (těm říkejme `default`, protože v nich nejsou extroviny z executablů) a potom se zavolá funkce `_run_executables`, uvnitř který (na ř. 188 v tomhle komitu) se jako vždycky zavolají additional feed dicts.

Co executables už nevrací nejsou additional feed dicty, ale seznamy coderů, který si tu držíme vedle v tom setu feedables.

Opravil se tim mimochodem latentní bug, kterej by se projevil v tom, že kdybys plnil placeholder z executablu, přepsal by se feed dictem z model partu.